spiceaidocs/docs/sdks/golang/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Go SDK'
+description: 'Connect to spice using spice go SDK'
+pagination_prev: null
+pagination_next: null
+---
+
+# Golang SDK for Spice.ai
+
+https://github.com/spiceai/gospice
+
+### Install
+
+```shell
+go get github.com/spiceai/gospice/v6
+```
+
+### Connect to spice runtime
+
+Import the package:
+
+```go
+import "github.com/spiceai/gospice/v6"
+```
+
+Create a `SpiceClient` using default configuration:
+
+```go
+spice := NewSpiceClient()
+defer spice.Close()
+```
+
+Or pass custom flight address:
+
+```go
+if err := spice.Init(
+    spice.WithFlightAddress("grpc://localhost:50052")
```

Comment:
Can we use a non `localhost` host. It's not immediately clear that this isn't the default path.

Reply:
Same for all python, js, rust.